bob/bob_test.rb

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'bob'
+
+class BobTest < Minitest::Test
+  def bob
+    ::Bob.new
+  end
+
+  def feedback(text)
+    "Bob hears #{text.inspect}, and.."
+  end
+
+  def test_stating_something
+    remark = 'Tom-ay-to, tom-aaaah-to.'
+    assert_equal 'Whatever.', bob.hey(remark), feedback(remark)
+  end
+
+  def test_shouting
+    remark = 'WATCH OUT!'
+    assert_equal 'Whoa, chill out!', bob.hey(remark), feedback(remark)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.